src/components/ContentHolder/ContentHolderComponent.js

```diff
@@ -1,71 +1,67 @@
-import React, { useContext, useState, useCallback, useMemo } from 'react';
+import React, {  useState, useCallback } from 'react';
 import TabComponent from '../TabComponent/TabComponent';
 import MovieListComponent from '../MovieList/MovieListComponent';
 import { ErrorBoundary } from '../ErrorBoundary/ErrorBoundary';
-import { Context } from '../../contextProvider';
+import { HeaderComponent } from '../HeaderComponent/HeaderComponent';
+import MovieDetails from '../MovieDetails/MovieDetailsComponent';
 import { TABS, DEFAULT_SORT_BY, DEFAULT_FILTER_BY } from '../../consts/constants';
+import { store, getMoviesActionAsync } from '../../store/store';
+import { connect } from 'react-redux';
```

Comment:
let's add import from libraries to the top of imports list)
move it after import react))